src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -248,6 +248,18 @@ public int MaxNumRecentFiles
         public bool OpenFileInManualExecutionMode { get; set; }
 
         /// <summary>
+        /// Indicates (if any) which namespaces should not be displayed in the Dynamo node library.
+        /// String format: "[library name]:[fully qualified namespace]"
+        /// </summary>
+        public List<string> NamespacesToExcludeFromLibrary { get; set; }
+
+        /// <summary>
+        /// True if the NamespacesToExcludeFromLibrary element is found in DynamoSettings.xml.
+        /// </summary>
+        [XmlIgnore]
+        public bool NamespacesToExcludeFromLibrarySpecified { get; set; }
```

Comment:
Can't we derive this property from NamespacesToExcludeFromLibrary ? If the count is more than zero then this is true?